起動時に出る警告メッセージを消す

PS D:\OneDrive\交通系\src\ICCardManager> dotnet run --project .\src\ICCardManager\
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Views\Dialogs\IncompleteBusStopDialog.xaml.cs(60,2
1): warning CS4014: この呼び出しを待たないため、現在のメソッドの実行は、呼び出しが完了するまで続行します。呼び出しの結果に 'await' 演算子を適用することを検討してください。 [
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
PS D:\OneDrive\交通系\src\ICCardManager>

--- a/ICCardManager/src/ICCardManager/Views/Dialogs/IncompleteBusStopDialog.xaml.cs
+++ b/ICCardManager/src/ICCardManager/Views/Dialogs/IncompleteBusStopDialog.xaml.cs
@@ -57,11 +57,11 @@ namespace ICCardManager.Views.Dialogs
                 if (updatedItem != null && !updatedItem.Summary.Contains("★"))
                 {
                     // 摘要が完全に更新された場合：ハイライト→2秒後に一覧から削除
-                    Dispatcher.InvokeAsync(() =>
+                    await Dispatcher.InvokeAsync(() =>
                     {
                         DataGridHighlightHelper.HighlightRow(BusStopDataGrid, updatedItem, 2.0, () =>
                         {
-                            Dispatcher.InvokeAsync(async () => await _viewModel.InitializeAsync());
+                            _ = Dispatcher.InvokeAsync(async () => await _viewModel.InitializeAsync());
                         });
                     }, DispatcherPriority.ContextIdle);
                 }

